managed/src/main/java/com/yugabyte/yw/common/PlacementInfoUtil.java

```diff
@@ -1743,29 +1748,48 @@ public static boolean isMultiAZ(Provider provider) {
     return azToConfig;
   }
 
-  public static String getKubernetesNamespace(String nodePrefix, String azName) {
-    return String.format("%s-%s", nodePrefix, azName);
+  public static String getKubernetesNamespace(String nodePrefix,
+                                              String azName, Map<String, String> azConfig) {
+    boolean isMultiAZ = azName == null ? false : true;
```

Comment:
minor : `boolean isMultiAZ = (azname == null)` is sufficient

Reply:
Haha, true. Changed it to `boolean isMultiAZ = (azName != null)`